che-mount/entrypoint.sh

```diff
@@ -93,45 +125,71 @@ trap 'stop_sync' SIGHUP SIGTERM SIGINT
 init_logging
 init_global_variables
 parse_command_line "$@"
-status=$?
-if [ $status -ne 0 ]; then
-    exit 1
+[ "$(ls -A /mnthost )" ] && EMPTY=false || EMPTY=true
+if [ $EMPTY = false ]; then
+    warn "(${CHE_MINI_PRODUCT_NAME} mount): Local folder is not empty. Are you sure?[Y/n]"
+    read yn
```

Comment:
we should check if we're running docker in interactive mode, to be sure we can read data from the user